frontend/config/webpack.config.prod.js

```diff
@@ -16,7 +16,15 @@ const getClientEnvironment = require('./env');
 
 // Webpack uses `publicPath` to determine where the app is being served from.
 // It requires a trailing slash, or the file assets will get an incorrect path.
-const publicPath = paths.servedPath;
+
+// This assigns '/' by default and breaks Django Webpack Loader
+//const publicPath = paths.servedPath;
+
+// setting it to '/static/{BUNDLE_DIR_NAME}/', where `BUNDLE_DIR_NAME` is the
+// same as defined figures/settings.py
+// We follow the standard folder naming scheme used for Django reusable apps
+const publicPath = '/static/figures/';
```

Comment:
@grozdanowski I just hardcoded this for now to fix the publicPath in  `webpack-stats.json` I didn't want to spend the time *right now* to see if I can fix it by finding out where process.env.PUBLIC_URL gets assigned and exploring the webpack config and build scripts. Another time.

Reply:
@johnbaldwin I see. I don't see a problem for that :)